### PR TITLE
Renames store to NamespaceStore

### PR DESCRIPTION
**Pull request by PGuimarais**
- renames store to NamespaceStore

Bamboo Build:
http://builds.cask.co/browse/CDAP-DRC4691

### CODE REVIEW

**Comment by ajainarayanan** [comment]
@PGuimarais If `NamespaceStore` is being exported as `NamespaceStore` we don't need to do 

```
import {default as NamespaceStore} from 'services/NamespaceStore'
```

We could just do 

```
import NamespaceStore from 'services/NamespaceStore'
```

**Comment by ajainarayanan** [comment]
Accidentally closed the PR. Sorry about that. My bad.

File: cdap-ui/app/cdap/cdap.js [hunk 1]
```diff
@@ -17,5 +17,8 @@
 import T from 'i18n-react';
 // Initialize i18n
 T.setTexts(require('./text/text-en.yaml'));
-
 require('./main.js');
+var PlusButton = require('../cdap/components/PlusButton').default;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/cdap.js]: Do we need this here? Where did this come from?

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**